src/installer/pkg/packaging/rpm/package.targets

```diff
@@ -315,84 +318,31 @@
 
   </Target>
   
-  <Target Name="GenerateRuntimeDependenciesRpm">
-    <PropertyGroup>
-      <RpmPackageName>$(RuntimeDependenciesRpmPkgName)</RpmPackageName>
-      <RpmPackageVersion>$(RuntimeDepsRpmPackageVersion)</RpmPackageVersion>
-      <RpmFile>$(DotnetRuntimeDependenciesPackageInstallerFile)</RpmFile>
-      <ConfigJsonName>dotnet-runtime-deps-rpm_config_$(OutputRid).json</ConfigJsonName>
-      <ConfigJsonFile>$(rpmPackagingConfigPath)$(ConfigJsonName)</ConfigJsonFile>
-      <RpmIntermediatesDir>$(PackagesIntermediateDir)$(RpmPackageName)/$(RpmPackageVersion)</RpmIntermediatesDir>
-    </PropertyGroup>
+  <Target Name="GenerateRuntimeDependenciesRpms">
 
     <PropertyGroup>
-      <RpmLayoutDirectory>$(RpmIntermediatesDir)/rpmLayoutDirectory/</RpmLayoutDirectory>
-      <RpmLayoutPackageRoot>$(RpmLayoutDirectory)package_root</RpmLayoutPackageRoot>
-      <RpmLayoutDocs>$(RpmLayoutDirectory)docs</RpmLayoutDocs>
-      <RpmLayoutTemplates>$(RpmLayoutDirectory)templates</RpmLayoutTemplates> <!-- Copyright, Changelog -->
+      <SharedProperties>rpmPackagingConfigPath=$(rpmPackagingConfigPath);DotnetRuntimeDependenciesPackageInstallerFile=$(DotnetRuntimeDependenciesPackageInstallerFile);OutputRid=$(OutputRid);RuntimeDependenciesRpmPkgName=$(RuntimeDependenciesRpmPkgName);RuntimeDepsRpmPackageVersion=$(RuntimeDepsRpmPackageVersion);RuntimeDepsRpmPackageRelease=$(RuntimeDepsRpmPackageRelease);TemplatesDir=$(TemplatesDir)</SharedProperties>
```

Comment:
Please split this onto multiple lines--I think newlines are ok, but if not, repeatedly appending to the prop is a typical way to do it.

Reply:
Will do.